internal/core/workers/operation_execution_worker/operation_execution_worker_test.go

```diff
@@ -72,6 +72,7 @@ func TestSchedulerOperationsExecutionLoop(t *testing.T) {
 		workerService := NewOperationExecutionWorker(scheduler, workers.ProvideWorkerOptions(operationManager, executors, nil, nil))
 
 		operationManager.EXPECT().NextSchedulerOperation(gomock.Any(), expectedOperation.SchedulerName).Return(expectedOperation, operationDefinition, nil)
+		operationManager.EXPECT().AppendOperationEventToExecutionHistory(gomock.Any(), expectedOperation, gomock.Any()).AnyTimes()
```

Comment:
@arthur29 WDYT about asserting the event message and the number of calls in these tests? we are using too permissive arguments here in a sense we are not testing if we are updating the operation history correctly